src/main/java/uk/gov/hmcts/reform/hmc/constants/Constants.java

```diff
@@ -29,4 +29,5 @@ private Constants() {
     public static final Integer UN_NOTIFIED_HEARINGS_LIMIT = 1000;
     public static final Integer FIRST_PAGE = 0;
     public static final String PENDING = "PENDING";
+    public static final Integer HEARING_DETAILS_DURATION = 360;
```

Comment:
A slightly more meaningful name 😄 
```suggestion
    public static final Integer DAY_DURATION = 360;
```